src/apps/urls.ts

```diff
@@ -117,15 +118,28 @@ export const customAppAddUrl = customAppAddPath;
 export const appsListUrl = (params?: AppListUrlQueryParams) =>
   appsListPath + "?" + stringifyQs(params);
 
-export const appIframeUrl = (
+export const resolveAppIframeUrl = (
   appId: string,
   appUrl: string,
   shopDomainHost: string,
   params: AppDetailsUrlQueryParams,
 ) => {
   const iframeContextQueryString = `?${stringifyQs(
-    { domain: shopDomainHost, id: appId, ...params },
+    {
+      /**
+       * @deprecated - domain will be removed in favor of saleorApiUrl.
+       * Current hostname (used as domain) can be extracted from full URL
+       *
+       * Difference will be:
+       * shop.saleor.cloud -> https://shop.saleor.cloud/graphql/
+       */
+      domain: shopDomainHost,
+      saleorApiUrl: getApiUrl(),
```

Comment:
`shopDomainHost` and `getApiUrl()` should match, correct? We can even use `getApiUrl()` to generate `shopDomainHost`.
I don't understand why there's such a different approach to obtaining these two values – one is passed as parameter and the other is imported from `@saleor/config`.

Reply:
`getApiUrl` is static on the build time so I don't see a reason it should be passed as a parameter

`shopDomainHost` will not match `getApiUrl` because one is a hostname, other is a full domain. Hence we pass both of them to iframe. Yes we can resolve hostman from saleorApiUrl but we want to keep compatibility for now